### PR TITLE
Added vue-directive-tooltip (vue.js v2)

### PR DESCRIPTION
**Pull request by hekigan**
Also added a “shield” to get a quick visual indication of Vue.js version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -507,6 +507,7 @@ Tooltips / popovers
 
  - [v-tooltip](https://github.com/Akryum/vue-tooltip) - Easy tooltips with Vue 2.x.
  - [vue-popper-component](https://github.com/antongorodezkiy/vue-popper-component) - Popper.js directive for Vue.js.
+ - [vue-directive-tooltip](https://github.com/hekigan/vue-directive-tooltip) - Simple, flexible tooltip directive (based on Popper.js) [![Vue.js](https://img.shields.io/badge/vue-2-green.svg)](https://vuejs.org)
```
Comment (kazupon) [inline README.md]: Please remove badge.

**Comment by hekigan** [comment]
Hi,
on the link below, it looks like adding badge is going to be the rule.
That's why I added it.

https://github.com/vuejs/awesome-vue/issues/936

**Comment by kazupon** [comment]
ah, sorry.

**Comment by hekigan** [comment]
No problem, thanks :)

**Pull request merged**